test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -69,6 +72,28 @@ function teardown() { _default_teardown ; }
   _should_receive_spam_at '/var/mail/localhost.localdomain/user1/.Junk/new/'
 }
 
+@test "(enabled + MARK_SPAM_AS_READ=1) should mark spam message as read" {
+  export CONTAINER_NAME=${CONTAINER2_NAME}
+
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_AMAVIS=1
+    --env ENABLE_SPAMASSASSIN=1
+    --env SA_SPAM_SUBJECT="SPAM: "
+    --env SPAMASSASSIN_SPAM_TO_INBOX=1
+    --env MOVE_SPAM_TO_JUNK=0
```

Comment:
Drop this, and rely on the implicit functionality, then we're just testing for `MARK_SPAM_AS_READ=1`:

```suggestion
```